aaa.py

```diff
@@ -0,0 +1,140 @@
+import binascii
+from tkinter import Tk, filedialog, Label, StringVar, ttk
```

Comment:
lepiej importować rzeczy w oddzielnych liniach i w porządku leksykograficznym:
```suggestion
from tkinter import (
    filedialog, 
    Label, 
    StringVar, 
    Tk,
    ttk,
)
```